exporters/SimpleEval.py

```diff
@@ -24,6 +24,7 @@
 import string
 
 from decimal import Decimal
+from others.py3compat import *
```

Comment:
Can we avoid * imports?

Reply:
Yes:
from others.py3compat import long, integer_types